doc/single-nic-vm-host-with-vlans.md

```diff
@@ -0,0 +1,270 @@
+# How to configure a virtual machine host with a single network interface and three VLANs
```

Comment:
This new document should also be linked from the "How-to guides" menu, by adding it to the `toctree` in `doc/howto.md`.